server/src/domain/repositories/storage.repository.ts

```diff
@@ -53,6 +53,7 @@ export interface IStorageRepository {
   readdir(folder: string): Promise<string[]>;
   stat(filepath: string): Promise<Stats>;
   crawl(crawlOptions: CrawlOptionsDto): Promise<string[]>;
+  walk(crawlOptions: CrawlOptionsDto): AsyncGenerator<string>;
```

Comment:
Why do we need crawl still?

Reply:
We don't, but the audit service still uses it. Should I change that to also use walk?

Reply:
Oh I see. We can do that in another pull request I think.